### PR TITLE
make a histogram

### PR DESCRIPTION
**Pull request by ateucher**
WIP - do not merge

### CODE REVIEW

**Comment by ateucher** [comment]
@BonnieJRobert not ready for review but just letting you know I'm working on it

File: andy-file-1.R [hunk 1]
```diff
@@ -1 +1,3 @@
-numbers <- rnorm(1000)
\ No newline at end of file
+numbers <- rnorm(1000)
```
Comment (BonnieJRobert) [inline andy-file-1.R:1]: try rpois()?
```suggestion
numbers <- rpois(10)
```

**Review by BonnieJRobert — CHANGES_REQUESTED** [review]
!!

**Review by BonnieJRobert — APPROVED** [review]
All looks great, review approved.

**Pull request merged**